README.md

```diff
@@ -20,37 +32,78 @@ With Drone Fly, the setup gets modified as shown in the diagram below. The only
+		   --env APIARY_KAFKA_TOPIC_NAME="dronefly" \
+		         expediagroup/drone-fly-app:<image-version>
+
+Then [Drone Fly Terraform](https://github.com/ExpediaGroup/apiary-drone-fly) module can be used to install your Docker image in a Kubernetes container.
 
-The Drone Fly image will be used as a base image by downstream projects which need a Hive Listener.
 
-Drone Fly uses the Jib plugin which will build a docker image during the `package` phase. The image can also be built directly:
+### Using Uber Jar
 
-    mvn compile jib:dockerBuild -pl drone-fly-app
-    
-# Running DroneFly
+Drone Fly build also produces an [uber jar](https://mvnrepository.com/artifact/com.expediagroup/drone-fly-app) so it can be started as a stand-alone Java service.
+
+#### Running DroneFly Jar
 
 	java -Dloader.path=lib/ -jar drone-fly-app-<version>-exec.jar \
 		--apiary.bootstrap.servers=localhost:9092 \
 		--apiary.kafka.topic.name=apiary \
-		--apiary.listener.list="com.expediagroup.sampleListener1,com.expediagroup.sampleListener2"
-	
-# Running DroneFly Docker image
-
-	docker run --env APIARY_BOOTSTRAP_SERVERS="localhost:9092" \
-			   --env APIARY_LISTENER_LIST="com.expediagroup.sampleListener1,com.expediagroup.sampleListener2" \
-			   --env APIARY_KAFKA_TOPIC_NAME="dronefly" \
-			   expediagroup/drone-fly-app:<image-version>
+		--apiary.listener.list="com.expediagroup.sampleListener1,com.expediagroup.sampleListener2"	
 	
 The properties `instance.name`, `apiary.bootstrap.servers`, `apiary.kafka.topic.name` and `apiary.listener.list` can also be provided in the spring properties file.
 	
 	java -Dloader.path=lib/ -jar drone-fly-app-<version>-exec.jar --spring.config.location=file:///dronefly.properties
 	
-# Terraform
+The parameter `-Dloader-path` is the path where Drone Fly will search for configured HMS listeners.
+
+## Configuring Drone Fly
+
+### Drone Fly configuration reference
+The table below describes all the available configuration values for Drone Fly.
+
+| Name | Description | Type | Default | Required |
+|------|-------------|------|---------|:--------:|
+| apiary.bootstrap.servers | Kafka bootstrap servers that receive Hive metastore events. | `string` | n/a | yes |
+| apiary.kafka.topic.name | Kafka topic name that receive Hive metastore events. | `string` | n/a | yes |
+| apiary.listener.list | Comma separated list of Hive metastore listeners to load from classpath. eg. `com.expedia.HMSListener1,com.expedia.HMSListener2` | `string` | `"com.expediagroup.dataplatform.dronefly.app.service.listener.LoggingMetastoreListener"` | no |
+| instance.name | Instance name for a Drone Fly instance. `instance.name` is also used to derive Kafka consumer group. Therefore, in a multi-instance deployment, unique `instance.name` for each Drone Fly instance needs to be provided to avoid all instances ending up in same Kafka consumer group. | `string` | `drone-fly` | no |
```

Comment:
```suggestion
| instance.name | Instance name for a Drone Fly instance. `instance.name` is also used to derive Kafka consumer group. Therefore, in a multi-instance deployment, unique `instance.name` for each Drone Fly instance needs to be provided to avoid all instances ending up in the same Kafka consumer group. | `string` | `drone-fly` | no |
```